src/combobox/combobox.component.ts

```diff
@@ -691,9 +691,8 @@ export class ComboBox implements OnChanges, AfterViewInit, AfterContentInit, OnD
 		});
 		this.view.items = this.items;
 		this.updatePills();
-		// clearSelected can only fire on type=multi
```

Comment:
This is false, this function was being called when clear input (x) is clicked. Previously would set the combobox selected element to an empty array which is incorrect for `single`.

Now it sets it to undefined which the dropdown-list emits on clear.